Quintet-iOS/View/CalendarView.swift

```diff
@@ -0,0 +1,157 @@
+                    .foregroundColor(.black)
+                    .font(.system(size: 15))
+            }
+            .padding(.bottom, 17)
+            
+            HStack(spacing: 2) {
+                
+                ForEach(days, id: \.self) { day in
+                    
+                    Text(day)
+                        .fontWeight(.medium)
+                        .padding(.horizontal)
+                        .foregroundColor(Color("DarkGray"))
+                    
+                }
+            }
+            
+            Divider()
+                .padding(.horizontal)
+            
+            let columns = Array(repeating: GridItem(.fixed(41)), count: 7)
+            LazyVGrid(columns: columns, spacing: 9) {
+                ForEach(extractDate()) { value in
+                    if value.day != -1 {
+                        CardView(value: value)
+                            .background (
+                                Circle()
+                                    .frame(width: 40, height: 40)
+                                    .foregroundColor(Color("DarkQ"))
+                                    .opacity(isSameDay(date1: value.date, date2: currentDate) ? 1 : 0)
+                            )
+                            .onTapGesture {
+                                currentDate = value.date
+                            }
+                    } else {
+                        Color.clear
+                    }
+                }
+            }
+            .fontWeight(.light)
+        }
+    
+        if let task = coreDataViewModel.getRecordMetaData(selectedYear: viewModel.selectedYear, selectedMonth: viewModel.selectedMonth).first (where: { task in
+                return isSameDay(date1: task.date, date2: currentDate)
+            }) {
+                VStack {
+                    Text("오늘의 5요소")
+                        .font(.system(size: 23))
+                        .fontWeight(.semibold)
+                        .padding(.trailing, 190)
+                        .padding(.bottom, 20)
+                        .padding(.top, 10)
+                        .foregroundColor(Color.black)
+                    
+                    ForEach(task.records) { record in
+                        recordCard(icon: record.icon, title: record.title, subtitle: record.subtitle)
+                    }
+                }
+                .onChange(of: currentMonth) { newValue in
+                    //updating Month
+                    currentDate = getCurrentMonth()
+                }
+            }
+    }
+        
+    @ViewBuilder
+    func CardView(value: DateValue) -> some View {
+        ZStack {
+            if value.day != -1 {
+                if let task = coreDataViewModel.getRecordMetaData(selectedYear: viewModel.selectedYear, selectedMonth: viewModel.selectedMonth).first (where: { task in
+                    return isSameDay(date1: task.date, date2: value.date)
+                }) {
```

Comment:
```suggestion
if value.day != -1 {
    if let task = coreDataViewModel
        .getRecordMetaData(
            selectedYear: viewModel.selectedYear,
            selectedMonth: viewModel.selectedMonth
        )
        .first(where: { task in
            return isSameDay(date1: task.date, date2: value.date)
        }) {
```
과 같이 작성하면 가독성이 더 좋아질 것 같습니다!!